pyanaconda/modules/common/structures/user.py

```diff
@@ -59,17 +60,32 @@ def name(self) -> Str:
     def name(self, name: Str):
         self._name = name
 
+    @property
+    def uid_mode(self) -> Str:
+        """Mode of UID.
+
+        Contains a string describing the mode of the user's UID: Use the value or default.
+
+        Possible values are: "ID_MODE_USE_VALUE" and "ID_MODE_USE_DEFAULT".
```

Comment:
Could you please write the items on lines below one on each line. It's more consistent with how the code is written.

Reply:
Thanks, sounds good, will do.